app/domain/entities/account/account.go

```diff
@@ -0,0 +1,71 @@
+package account
+
+import (
+	"errors"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/jpgsaraceni/suricate-bank/app/cpf"
+	"github.com/jpgsaraceni/suricate-bank/app/hash"
+)
+
+type (
+	AccountId uuid.UUID
+	Money     int
+)
+
+type Account struct {
+	Id        AccountId
+	Name      string
+	Cpf       string
+	Secret    string
```

Comment:
Não vai usar os tipos customizados aqui?
```suggestion
	Cpf       cpf.Cpf
	Secret    hash.Secret
```